libguard/guard_interface.cpp

```diff
@@ -315,6 +317,7 @@ void clearAll()
     {
         for_each_guard(file, pos, existGuard)
         {
+		    printf("pos %d, lastPos %d\n", pos, lastPos);
```

Comment:
is it required? you said added for debug purpose and going to remove but still seeing the trace.